package.json

```diff
@@ -61,7 +61,7 @@
     "init_example": "node scripts/init_example.js",
     "execute_integration_tests": "jest --maxWorkers 1 --config test_integration/jest.config.json",
     "unlink_example": "cd ./example && npm unlink @sap-webide/webide-client-tools",
-    "docs": "shx rm -rf docs/web/html_docs && typedoc --tsconfig tsdocsconfig.json --out docs/web/html_docs --module commonjs --name WebideClientTools --excludeExternals --includeDeclarations",
+    "docs": "shx rm -rf docs/web/html_docs && typedoc api.d.ts --tsconfig tsdocsconfig.json --out docs/web/html_docs --name WebideClientTools --excludeNotDocumented",
```

Comment:
why `excludeNotDocumented` it does not make much sense to exclude
items from api.d.ts that do not have a block comment `/** **/`

Reply:
np, I'll remove that flag in next change